src/lib/pages/module-details/components/ModuleTop.tsx

```diff
@@ -32,11 +32,10 @@ const baseTextStyle: TextProps = {
   whiteSpace: "nowrap",
 };
 
-export const ModuleTop = ({ moduleData, isVerified }: ModuleTopProps) => {
+const ModuleCta = ({ moduleData }: { moduleData: IndexedModule }) => {
```

Comment:
We can pass `moduleAddress` from `ModuleTop`, hence we no need to calculate it again.